### PR TITLE
Removing deprecated customer id from createCustomerAddress mutation

### PR DESCRIPTION
**Pull request by VinothKumar361**
## Purpose of this pull request

This pull request (PR) removes deprecated customer id from createCustomerAddress mutation. source code
(https://github.com/magento/magento2/blob/5f3b86ab4bd3e3b94e65429fed33f748d29c1bbe/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L103)
(https://github.com/magento/magento2/blob/2.3/app/code/Magento/CustomerGraphQl/etc/schema.graphqls)

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/graphql/mutations/create-customer-address.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/_includes/graphql/customer-address-input.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Attribute |  Data Type | Description
 `country_code` | String | The customer's country
 `country_id` | String | Deprecated. Use `country_code` instead. The customer's country
 `custom_attributes` | [CustomerAddressAttributeInput](#customerAddressAttributeInput) | Deprecated. Not applicable for GraphQL
-`customer_id` | Int | The customer ID
+`customer_id` | Int | Deprecated. This attribute is not applicable for GraphQL. The ID assigned to the customer
```
Comment (rogyar) [inline src/_includes/graphql/customer-address-input.md]: Hi @VinothKumar361. I'm not sure we still have this field. You can doublecheck the [schema](https://github.com/magento/magento2/blob/5f3b86ab4bd3e3b94e65429fed33f748d29c1bbe/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L22) for the recent 2.3 release

Comment (VinothKumar361) [reply src/_includes/graphql/customer-address-input.md]: Hi @rogyar : on seeing this 
[schema ](https://github.com/magento/magento2/blob/5f3b86ab4bd3e3b94e65429fed33f748d29c1bbe/app/code/Magento/CustomerGraphQl/etc/schema.graphqls#L22) the CustomerAddressInput Object has some unwanted attributes in the  [docs](https://devdocs.magento.com/guides/v2.3/graphql/mutations/create-customer-address.html#customerAddressInput).i removed them and commited the changes.please review it now.

**Comment by keharper** [comment]
@rogyar Would you review this PR again?

File: src/_includes/graphql/customer-address-input.md (hunk 1 above)
Comment (rogyar) [reply src/_includes/graphql/customer-address-input.md]: Thank you!

**Pull request merged**